internal/lm/nvml.go

```diff
@@ -193,3 +201,59 @@ func isMPSCapable(manager resource.Manager) (bool, error) {
 	}
 	return true, nil
 }
+
+func newGPUModeLabeler(manager resource.Manager) (Labeler, error) {
```

Comment:
let's add a resonable docstring to the constructor.

Also, we could consider using:
```suggestion
func newGPUModeLabeler(devices []resource.Device) (Labeler, error) {
```
instead since we already have a slice of devices as the callsite.

Reply:
Used devices from the calling function